src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,93 @@
+package duke.command;
+
+public abstract class Command {
+    /**
+     * Checks if the command is meant to delete the task(s)
+     *
+     * @param command The input command
+     * @return Returns true if the command starts with 'delete' no matter it is uppercase or lowercase or mixed, false otherwise
+     */
+    public static boolean isCommandDelete(String command) {
+        return command.toLowerCase().startsWith("delete");
+    }
+
+    /**
+     * Checks if the command is meantto delete all the tasks stored
+     *
+     * @param word The second word in the command
+     * @return Returns true if the second word in the command is equivalent to 'all' or '*' no matter it is uppercaseor lowercase or mixed, false otherwise
```

Comment:
This line may be too long (>120 characters): https://se-education.org/guides/conventions/java/basic.html#layout
Applies to any lines >120 characters.